R/ml_model_als.R

```diff
@@ -0,0 +1,16 @@
+new_ml_model_als <- function(pipeline_model,
+                             formula,
+                             dataset,
+                             label_col,
+                             features_col) {
+ m <- new_ml_model_regression(
```

Comment:
We shouldn't inherit ml_model_regression here since it's not a regression model and there are no `label_col`, `features_col`, etc.

Reply:
is it better to create a new_model_recommendation?

Reply:
yeah I think that's the right thing to do

Reply:
Ok, I'll work on it